kickstarts/partials/post/python_modules.ks.erb

```diff
@@ -37,6 +36,7 @@ azure-mgmt-cosmosdb==0.5.2
 azure-mgmt-devtestlabs==3.0.0
 azure-mgmt-dns==2.1.0
 azure-mgmt-hdinsight==0.1.0
+azure-mgmt-iothub==0.7.0
```

Comment:
So something like this, I'm not sure we need. I don't think we'll be managing IOT in Azure.